.evergreen/generate_mongosh_tasks.js

```diff
@@ -0,0 +1,66 @@
+const mongoshPackages = ['arg-parser',
+	'async-rewriter2',
+	'autocomplete',
+	'browser-repl',
+	'browser-runtime-core',
+	'browser-runtime-electron',
+	'build',
+	'cli-repl',
+	'connectivity-tests',
+	'editor',
+	'errors',
+	'history',
+	'i18n',
+	'java-shell',
+	'js-multiline-to-singleline',
+	'logging',
+	'mongosh',
+	'node-runtime-worker-thread',
+	'service-provider-core',
+	'service-provider-server',
+	'shell-api',
+	'shell-evaluator',
+	'snippet-manager',
+	'types',
+	'docker-build-scripts']
```

Comment:
- Keeping this as a hardcoded list isn’t going to be very maintainable; if you go with this approach, I would at least add a check somewhere that compares this list against the actual list
- There’s only a subset of these packages that is _really_ affected by driver changes (`lerna ls --scope @mongosh/service-provider-server --scope @mongosh/connectivity-tests --include-dependents` in mongosh)

Reply:
- I did consider this, but I did make the assumption that the package structure of mongosh doesn't change much.  Is that incorrect?  Adding a check would make sense, but I could also just generate them each time the script is built (clone mongosh and lerna list).
- My intention here was to break apart the existing task just by scope.  If you think that the existing task was testing more than was necessary, then we can reconsider 😄